src/helpers/byDisplayValue.ts

```diff
@@ -23,7 +23,10 @@ const getTextInputNodeByDisplayValue = (
       matches(value, nodeValue, normalizer, exact)
     );
   } catch (error) {
-    throw createLibraryNotSupportedError(error);
+    if (error instanceof Error) {
```

Comment:
Common theme in this PR. `catch` provides `error: unknown` in recent TS, so I've been trying to fix the code around, feel free to chime in and let me know if you have different opinion on the workarounds

Reply:
We should not silently ignore the non-`Error` values. It should be better to just analyse the `error` type inside the `createLibraryNotSupportedError`.